edgelet/iotedge/src/main.rs

```diff
@@ -64,6 +61,16 @@ fn run() -> Result<(), Error> {
                 .env("IOTEDGE_HOST")
                 .default_value(default_mgmt_uri),
         )
+        .arg(
```

Comment:
Why does this need to be configurable? Just for local dev? If so, might be useful to make it `#[cfg(debug_assertions)]`

Reply:
Yes, I've primarily been using it for local dev. I don't imagine any end users would have to manually set it.

Instead of customizing it via CLI flag, how about I read it from an `AZIOT_BIN` env var instead? In that case, should I hide it behind `#[cfg(debug_assertions)]`, or would it be fine to leave it active in release builds as well?

Reply:
Oh, better yet, I can just use [`option_env!`](https://doc.rust-lang.org/std/macro.option_env.html) instead. We already use it for `IOTEDGE_HOST`. No `#[cfg(debug_assertions)]` needed :smile: 

Sidenote, I didn't know `option_env!` was a thing - very cool!